plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarSpeech.java

```diff
@@ -125,18 +98,23 @@ public String getEventsAt(Map<String, EntityData> entities) {
 		List<CalendarEvent> events;
 		List<String> eventList = new ArrayList<>();
 		LocalDateTime chosenDate = LocalDateTime.of(entities.get("date").getDate(), ZERO);
-		if (!entities.get("date").getString().equals(TODAY) && !entities.get("date").getString().equals("tomorrow")
-				&& entities.get("eventyear") == null && entities.get("date").getDate()
+		if (entities.get("eventyear") == null && entities.get("date").getDate()
 				.isBefore(this.environment.getCurrentLocalDateTime().toLocalDate())) {
 			chosenDate = chosenDate.withYear(this.environment.getCurrentLocalDateTime().getYear() + 1);
 		}
 		events = this.logic.getEventsAt(chosenDate);
-		if (events.isEmpty()) {
-			return "No events found for the " + getDate(chosenDate) + " " + chosenDate.getYear() + ".";
-		}
 		for (CalendarEvent event : events) {
 			eventList.add(this.checkDate(chosenDate, event, false));
 		}
+		if (entities.get("date").getString().equals("today") || entities.get("date").getString().equals("tomorrow")) {
```

Comment:
create a String for entities.get("date") and then compare with the string. 
You should also use it later to insert it into your return value